File: Level 36/Classwork/Classwork.py
def multiply(a, b):
    return a * b

#Even or Odd
def even_or_odd(number):
    if number %2 == 0:
        return "Even"
    else: return "Odd"

#Convert a Number to a String!
def number_to_string(num):
    return str(num)

#Reversed Strings
def solution(string):
    return string[::-1]

#Return Negative
def make_negative( number ):
    if number > 0:
        return number * -1
    else: return number

#Opposite number
def opposite(number):
    return number * -1

#convert buleon yes to no
def bool_to_word(boolean):
    if boolean==True:
        return "Yes"
    else:
        return "No"
#sum
def positive_sum(arr):
    res = 0
    for i in arr:
        if i > 0:
            res += i
    return res
#repeat
def repeat_str(repeat, string):
    res = ""
    for i in range(repeat):
        res = res + string
    return res

#remove first and last
def remove_char(s):
    return s[1:-1]

def square_sum(numbers):
    result=0
    for i in numbers:
        result+=i**2
    return result

File: Level 83/Classwork/Classwork.js
const numbers = [5, 8, 12, 7, 15, 4, 10, 3];
let sum = 0;

for (let i = 0; i < numbers.length; i++) {
  sum += numbers[i];

  if (sum > 50) {
    console.log("შეკრება შეწყდა — ჯამი გადააჭარბა 50-ს");
    break;
  }
}

console.log("საბოლოო ჯამი:", sum);